### ISSUE TITLE
Mouse support for the BattleRingCommand_Beta Type-β please.

### ISSUE THREAD

**Issue by Greko5** [soryu-rmv/BattleRingCommands#1]
Hi  soryu-rmv, 
i love your ring menu (BattleRingCommand_Beta - Type-β) is it possible that you can make it compatible with the pc mouse?

Hope so, thanks for your very nice Plugins(Need it for MV, not for MZ)

**Comment by soryu-rmv** [comment]
Hi Greko5.

There are no technical problems to handle a ring with the mouse.
(Actually I did not implement that on purpose, and will do if some offers come up.)

But currently, I'm just working on a task to transfer the all materials here to my new coming website 
for further plugin publication and their management (for MV and MZ).
So, I'll put your offer into my task and update SoR_BattleRingCommand soon before MZ is released.
Give me a moment.

(I'm concurrently working on the original UI design for PartyCommandSelection 
whose architecture is almost same as the BattleRingCommand. 
Then I also should make it compatible with the mouse.)

**Comment by Greko5** [comment]
Thank you very much for taking my offer on your task list, i'm waiting for your update.
No problem take your time to complete your other tasks first.

**Comment by soryu-rmv** [comment]
OK, I just want to ask one question to update the script.
What kinds of behavior for the mouse do you need?

For example,
- rotate the ring by taking amount of scroll wheel,
- select the icon on the ring by click.

Actually, I've already implemented the former behavior.
Although, since I always use keyboard or gaming pad for the input,
I cannot figure out the obvious expected specification of mouse control for RPGMV.

**Comment by Greko5** [comment]
First sorry for the late reply
I only use the mouse i never use the keyboard or GamePad, in my game all works with the mouse only the battle ring menu doesn't work ^^

I mean: When you scroll with your mouse up, the ring menu goes to the next right icon and when you scroll down, it goes to the next left icon and after the scroll the right icon that you need is in front of you instead of the sword(best to understand look at picture 1).

After the scroll the right icon is in front of you instead of the sword, now you can click the left mouse button on it to come to the next window, where the generall window for skills, Items... is.   (best to understand - picture 2), here the Battle Ring tasks ends, and after that all works well because the generall window already works with the mouse.

Please look at the pictures, here you could understand it better(sry for bad explanation) ^^
https://www.directupload.net/file/d/5880/y76hnlzx_png.htm
https://www.directupload.net/file/d/5880/q2bv566z_png.htm

**Comment by soryu-rmv** [comment]
Thanks for your detailed explanation.
I now substitute a script on the repository just for Type-β (currently not announced for public).
It behaves like a ring menu provided by MOG_BattleHud in terms of mouse control based on the original structure of RPGMV.
Also some optimizations are applied.

Please check the new version to test whether it satisfies you.

**Comment by Greko5** [comment]
You are a Hero, works very nice thank you so much! :)

**Comment by soryu-rmv** [comment]
Sounds good!
I'll then update every version to apply the same behavior 
and release the new version. You may use the current Type-β script.
Thank you for your kind offer.

**Comment by Greko5** [comment]
Yes that's a good idea i think there are many people who would like to control your Battle ring menus with the mouse.
No problem but i have to thank you, especially that you have integrated it so fast, i must say that i really love your beta type ring menu, whished MV or MZ would have such Ring Menus/Battle Huds/Commands as default in their programm or would sell it as DLC's.
